app/(pages)/feedback/page.tsx

```
@@ -0,0 +1,49 @@
+import { JSX } from "react"
+import Link from "next/link";
+import {GoDotFill} from "react-icons/go";
+import PartnerForm from "@/components/PartnerForm";
+import {Mail, MapPin, Phone} from "lucide-react";
+import Image from "next/image";
+import {Metadata} from "next";
+import AppSettings from "@/Oceanoftech.Business/ConfigurationBusiness/AppSettings";
+import Map from "@/components/Map"
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)